### PR TITLE
Feature/application form modal ln

### PR DESCRIPTION
**Pull request by anewb87**
### 1. What changed?
Added an application form modal component and styled it.

### 2. Why is this change necessary?
Our application requires this form. We want users to submit applications. While there isn't yet any logic, we've set it up and it's ready to go.

### 3. What changed technically that may impact others?
N/A

### 4. How do we test it?
Wrote Cypress tests to confirm elements on the application

### CODE REVIEW

File: cypress/integration/application_form_spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+describe('Applicant Form Modal', () => {
+    it('Should display the form on the page', () => {
+        cy.visit('http://localhost:3000/application')
```
Comment (Fordo29) [inline cypress/integration/application_form_spec.js:3]: Well done on setting up the initial testing for what is showing on the page.

File: src/Components/App.js [hunk 2]
```diff
@@ -5,15 +5,17 @@ import Home from './Home';
...
+import ApplicationFormModal from './ApplicationFormModal';
+import DonationFormModal from './DonationFormModal';
 
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/"  element={<Home />} />
         <Route path="pet/:id" element={<Pet />} /> 
+        <Route path='application' element={<ApplicationFormModal />} />
```
Comment (Fordo29) [inline src/Components/App.js:17]: We will have to be mindful with these 2 routes when we link them to the appropriate pet_id

File: cypress/integration/application_form_spec.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            .get('h3')
+            .should('have.text', 'Contact Information')
+            .get('label')
+            .contains('Full Name')
+            .get('label')
+            .contains('Email')
+            .get('label')
+            .contains('Tell the owner')
+            .get('button')
+            .contains('Submit')
```
Comment (StevieAmb) [inline cypress/integration/application_form_spec.js:15]: This is an excellent start! Next steps will be to grab the input fields and check that they can be typed into!

File: src/Styles/base.scss [hunk 4]
```diff
@@ -12,10 +12,68 @@
...
+::placeholder {
+  font-family: inherit;
+}
+
+textarea {
+  resize: none;
+  width: 99%;
+}
+
+.modal {
```
Comment (Fordo29) [inline src/Styles/base.scss:38]: This was a great learning on how to control the styling for the application.  Feels much more organized.

File: src/Components/ApplicationFormModal.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+const ApplicationFormModal = () => {
+    const [formState, setFormState] = useState({
+        name: '',
+        email: '',
+        description: ''
+    })
+
+    return (
+        <div className='modal'>
```
Comment (StevieAmb) [inline src/Components/ApplicationFormModal.js:12]: Great work making this modal!

**Review by Fordo29 — APPROVED** [review]

File: src/Components/ApplicationFormModal.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+                                        ...formState,
+                                        description: e.target.value
+                                    })
+                                }
+                                required
+                            />
+                        </div>
+                    </div>
+                </section>
+                <button className="submit-button">Submit</button>
```
Comment (StevieAmb) [inline src/Components/ApplicationFormModal.js:71]: Next steps would be to add a function to this button onClick, that will POST the application.

**Review by StevieAmb — APPROVED** [review]

**Review by ectweitmann — COMMENTED** [review]
This looks good. The one thing I noted is that you used single quotes for all your JSX where I used double quotes for mine. Either way is fine-- it also doesn't effect functionality obviously-- but we should pick one way and stick to it for consistency.

**Pull request merged**